sagemaker-clarify/time_series_byom/time_series_bring_your_own_model.ipynb

```diff
@@ -0,0 +1,1019 @@
+    "# TimeSeries Bing Your Own Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52f85a11",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "This notebook's CI test result for us-west-2 is as follows. CI test results in other regions can be found at the end of the notebook. \n",
+    "\n",
+    "![This us-west-2 badge failed to load. Check your device's internet connectivity, otherwise the service is currently unavailable](https://prod.us-west-2.tcx-beacon.docs.aws.dev/sagemaker-nb/us-west-2/sagemaker-clarify|time_series_byom|time_series_bring_your_own_model.ipynb)\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5967044",
+   "metadata": {},
+   "source": [
+    "## Runtime\n",
+    "\n",
+    "This notebook takes approximately 30 minutes to run."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3683bcbd",
+   "metadata": {},
+   "source": [
+    "## Prerequisites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7efb226e",
+   "metadata": {},
+   "source": [
+    "### Install Mercury\n",
+    "\n",
+    "If not already installed, the following cell will install the `mercury` package in order to display the `analysis_config.json` and explainability job output within the notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "bc3ffd45",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[33mWARNING: Ignoring invalid distribution -otocore (/local/home/zicanl/.virtualenvs/venv/lib/python3.9/site-packages)\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[33mWARNING: dj-rest-auth 3.0.0 does not provide the extra 'with-social'\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[33mWARNING: Ignoring invalid distribution -otocore (/local/home/zicanl/.virtualenvs/venv/lib/python3.9/site-packages)\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
+      "awscli 1.31.7 requires botocore==1.33.7, but you have botocore 1.29.165 which is incompatible.\n",
+      "awscli 1.31.7 requires s3transfer<0.9.0,>=0.8.0, but you have s3transfer 0.6.2 which is incompatible.\n",
+      "sagemaker 2.214.3 requires boto3<2.0,>=1.33.3, but you have boto3 1.26.83 which is incompatible.\u001b[0m\u001b[31m\n",
+      "\u001b[0m"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install mercury -q"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1686b70e",
+   "metadata": {},
+   "source": [
+    "## Install Development SDK\n",
+    "\n",
+    "As of 13-12-23, time series explainability (TSX) is in closed beta. The changes to the SDK to support it are not merged into the `sagemaker-python-sdk` repository. Instead, we install a version of the PySDK that does support these changes directly via a wheel file. "
```

Comment:
The notebook no longer installs the SDK from a wheel file, please update this description.

Reply:
fixed

Reply:
The diff currently shows this is not fixed, can you confirm?